x/concentrated-liquidity/swaps.go

```diff
@@ -301,21 +320,37 @@ func (k Keeper) calcOutAmtGivenIn(ctx sdk.Context,
 			return writeCtx, sdk.Coin{}, sdk.Coin{}, sdk.Int{}, sdk.Dec{}, sdk.Dec{}, fmt.Errorf("could not convert next tick (%v) to nextSqrtPrice", nextTick)
 		}
 
+		// Charge fee
+		feeOnFullAmountRemainingIn := swapState.amountSpecifiedRemaining.Mul(swapFee)
+
+		// TODO: remove this print.
+		fmt.Println("sqrtPrice before swap step", swapState.sqrtPrice)
```

Comment:
Note: this and other removal TODOs are tracked in #4096

Reply:
Might be useful for debugging while we implement the swap in the other direction. Since the helpers are shared, changing them might break this kind of swap as well

Reply:
I don't mind removing, I can just re-add them as needed so let me know if you strongly feel about this

Reply:
removed now

Reply:
You could also leave them as commented prints like we do elsewhere if needed